sci-libs/openblas/files/openblas-0.3.20-fix-riscv.patch

```diff
@@ -0,0 +1,113 @@
+From 3fc52ebcfb80e01f753938fd314ca07b2c085767 Mon Sep 17 00:00:00 2001
```

Comment:
Please put a link to the Gentoo bug and upstream PR / commits at the top of the patch too.

Reply:
OK. I resovled it.